settings.gradle.kts

```diff
@@ -8,10 +8,6 @@ pluginManagement {
   }
 }
 
-plugins {
```

Comment:
can you add a comment here about the problem with adding toolchain support so we don't accidentally add it back in?